src/index.ts

```diff
@@ -749,7 +747,7 @@ export class Bigtable {
             currentRetryAttempt: 0,
             noResponseRetries: 0,
             objectMode: true,
-            shouldRetryFn: (Service as any).shouldRetryRequest_,
+            shouldRetryFn: r => [429, 500, 502, 503].indexOf(r.code) > -1,
```

Comment:
We should refactor this to use gax for handling retries, it's perfectly capable of doing it.  (I didn't read this code so it might be harder than it sounds)

Reply:
Hrumph.  The comments in the code point to here, which makes me think maybe we don't support retries natively here yet?
https://github.com/googleapis/gax-nodejs/blob/master/src/streamingCalls/streamDescriptor.ts#L55

Reply:
likewise, I care less about this logic being pulled in from `grpc-common`, and care more that we make sure there's one test that exercise the retry logic.